draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -3578,19 +3575,19 @@ Constants (see discussion below):
 5. c5 = c4^c2 in F
 
 Procedure:
-1.  r = x^c3
-2.  t = r * r * x
-3.  r = r * x
+1.  s = x^c3
+2.  t = s * s * x
+3.  s = s * x
 4.  b = t
 5.  c = c5
-6.  for k in (m, m - 1, ..., 2):
+6.  for k in (c1, c1 - 1, ..., 2):
 7.      for j in (1, 2, ..., k - 1):
```

Comment:
```suggestion
7.      for j in (1, 2, ..., k - 2):
```

Oops! This is my fault.

I just mechanically translated the code from the latest commit into Sage and it runs correctly after the above change.